src/pages/Messages.jsx

```diff
@@ -67,14 +69,14 @@ const Messages = () => {
   }, [outboxList]);
 
   const [boxType, setBoxType] = useState('inbox');
-  const [showForm, setShowForm] = useState(false);
+  const [showModal, setShowModal] = useState(false);
 
   return (
     <Box sx={{ display: 'grid', gridTemplateRows: '80px 1fr' }}>
       <Box sx={{ display: 'flex', padding: '20px 30px 10px' }}>
         <Button
           variant="contained"
-          onClick={() => setShowForm(!showForm)}
+          onClick={() => setShowModal(!showModal)}
           startIcon={<CreateIcon />}
           sx={{
```

Comment:
this full `sx` should be replaced with `color="secondary"` - will do the same, built in feature of `Button`